tableaudocumentapi/datasource.py

```diff
@@ -213,6 +213,10 @@ def version(self):
     def connections(self):
         return self._connections
 
+    def clear_repository_location(self):
+        tag = self._datasourceXML.find('./repository-location')
+        self._datasourceXML.remove(tag)
```

Comment:
This will fail if line 217 fails -- like if you call this method twice or it never had a `repository-location` tag.

Do we just want this to return silently instead of blowing up?